jbmc/src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -1655,6 +1658,32 @@ void java_bytecode_parsert::rinner_classes_attribute(
   }
 }
 
+/// Corresponds to the element_value structure
+/// Described in Java 8 specification 4.7.5
+/// https://docs.oracle.com/javase/specs/jvms/se7/html/jvms-4.html#jvms-4.7.5
+/// Parses the Exceptions attribute for the current method,
+/// which contains a table of the exceptions the method throws.
+void java_bytecode_parsert::rexceptions_attribute(methodt &method, const u4 &attribute_length)
+{
+  std::string name = method.name.c_str();
+  u2 number_of_exceptions = read_u2();
+  method.throws_exception_table.resize(number_of_exceptions);
+
+  const auto pool_entry_lambda = [this](u2 index) -> pool_entryt & {
```

Comment:
I think this `pool_entry_lambda` is making the code harder to read. Why not use `pool_entry(exception_index_table)` inline, instead of `pool_entry_lambda(exception_index_table)`?

Reply:
I've done this another way, so this is no longer required.